apps/judicial-system/backend/src/app/modules/police/police.service.ts

```diff
@@ -347,37 +350,24 @@ export class PoliceService {
 
           this.subpoenaStructure.parse(response)
 
-          const subpoenaToUpdate = await this.subpoenaService.findBySubpoenaId(
-            subpoenaId,
-          )
-
-          const serviceStatus = response.deliveredToLawyer
-            ? ServiceStatus.DEFENDER
-            : response.prosecutedConfirmedSubpoenaThroughIslandis
-            ? ServiceStatus.ELECTRONICALLY
-            : response.deliveredOnPaper || response.delivered === true
-            ? ServiceStatus.IN_PERSON
-            : response.acknowledged === false && response.delivered === false
-            ? ServiceStatus.FAILED
-            : // TODO: handle expired
-              undefined
-
-          if (serviceStatus === undefined) {
-            return subpoenaToUpdate
+          return {
```

Comment:
The police service simple returns the latest subpoena info from the police system. Subpoena updates are no longer initiated from there, but rather from the subpoena service when retrieving subpoenas.